api/app/services/playthroughs_service.py

```
@@ -0,0 +1,968 @@
+            )
+        except Exception as e:  # noqa: BLE001
+            self.db.rollback()
+            logger.error(f"Database error deleting playthrough: {e}")
+            raise OperationError("Failed to delete playthrough")
+
+    def bulk_playthrough_operations(
+        self, *, current_user: CurrentUser, bulk_request: PlaythroughBulkRequest
+    ):
+        if bulk_request.action == BulkAction.UPDATE_STATUS:
+            if not bulk_request.data or "status" not in bulk_request.data:
+                raise BadRequestError("Status is required for update_status action")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)